js/cryptocom.js

```diff
@@ -2317,6 +2349,30 @@ module.exports = class cryptocom extends Exchange {
         return rates;
     }
 
+    handleMarginModeAndParams (methodName, params = {}) {
+        /**
+         * @ignore
+         * @method
+         * @description marginMode specified by params["marginMode"], this.options["marginMode"], this.options["defaultMarginMode"], params["margin"] = true or this.options["defaultType"] = 'margin'
+         * @param {object} params extra parameters specific to the exchange api endpoint
+         * @returns {[string|undefined, object]} the marginMode in lowercase
+         */
+        const defaultType = this.safeString (this.options, 'defaultType');
+        const isMargin = this.safeValue (params, 'margin', false);
+        let marginMode = undefined;
+        [ marginMode, params ] = super.handleMarginModeAndParams (methodName, params);
```

Comment:
@Dan-krm, currently there's an issue because the python version is not recognizing the camelCase version of `handlemarginmodeandparams`, going to check it now 

![image](https://user-images.githubusercontent.com/43336371/186438256-69ee5325-3cc7-4f2e-a4a7-0d3bfbe3efa6.png)